Editor/Scripts/wind_forces_tutorial.py

```diff
@@ -0,0 +1,94 @@
+                In <b>Entity Inspector</b>, in the <b>PhysX Collider</b> component, set the <b>Shape</b> property to 
+                <strong><code style="font-size:14px;color:#E44C9A">"Box"</code></strong>.<br><br>
+                The wind force can only affect entities that are within this box collider shape. </p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Resize the wind provider", """<html><p style="font-size:13px">The box is quite small. Let's enlarge it so that it affects a larger area and so that we can easily position the cloth entity inside it.
+                <br><br>In <b>Entity Inspector</b> in the <b>PhysX Collider</b> component, set the <b>X</b>, <b>Y</b>, and <b>Z</b> components of the <b>Dimensions</b> property to 
+                <strong><code style="font-size:14px;color:#E44C9A">"text"</code></strong>.</p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Position the Entity", """<html><p style="font-size:13px"> Use the <b>Move</b> tool to position the wind provider entity in the level. 
+                Ensure that the bottom of the PhysX collider box is roughly on the ground plane, and that the collider is in the camera's view frustum.</p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Add a PhysX Force Region component", """<html><p style="font-size:13px"> 
+                <br><br>Our wind provider entity has a Tag that identifies it as a local wind provider and a box collider that defines a volume for the wind force. We need to add a <b>PhysX Force Region</b> that defines the wind force.<br><br>
+                In <b>Entity Inspector</b>, choose <b>Add Component</b>, and select <b>PhysX Force Region</b> from the component list. </p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Configure the wind force direction", """<html><p style="font-size:13px"> 
+                We need to specify a direction for the wind force. <br><br>In <b>Entity Inspector</b>, in the <b>PhysX Force Region</b> 
+                component, click the <b>+</b> button to add a force. <br><br>Set the <b>X</b> value of the <b>Direction</b> property to -1.0. Set the <b>Y</b> 
+                value of the <b>Direction</b> property to -8.0. Set the <b>Z</b> value of the <b>Direction</b> property to 0.0. <br><br> 
+                These value create a slightly off axis horizontal wind direction. Notice that the collider box has blue cones that display the direction of the wind force. </p></html>""", "InspectorMainWindow"))        
+        self.add_step(TutorialStep("Configure the wind force magnitude", """<html><p style="font-size:13px"> 
+                We can specify the strength of the wind force with the magnitude property.
+                <br><br>In Entity Inspector, in the PhysX Force Region component, set Magnitude property to 8.0. </p></html>""", "InspectorMainWindow"))                
```

Comment:
Highlight the Magnitude property

```suggestion
        self.add_step(TutorialStep("Configure the wind force magnitude", """
                <html><p style="font-size:13px">You can specify the strength of 
                the wind force with the magnitude property.<br><br>In <b>Entity 
                Inspector</b>, in the <b>PhysX Force Region</b> component, set 
                the <b>Magnitude</b> property to <strong>
                <code style="font-size:14px;color:#E44C9A">8.0</code></strong>.
                </p></html>""", 
                {"text": "Magnitude", "type": QtWidgets.QFrame}))
```